SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -156,6 +166,11 @@ public let DEFAULT_DISK_CACHE_MAX_SIZE_BYTES: Int64 = 25 * 1024 * 1024
      */
     @objc public var sessionSamplingRatio: Double = 1.0
 
+    /**
+     Set the time interval between batch span sampling in seconds
+     */
+    @objc public var spanSchedulingDelay: TimeInterval = 5.0
```

Comment:
A few alternative naming ideas: `bspScheduleDelay`, `batchSpanProcessorScheduleDelay` to hint that it's related to OTel's batch span processor? For example in the OTel spec the environment variable corresponding to this option is named `OTEL_BSP_SCHEDULE_DELAY`